### ISSUE TITLE
Can not open plugin manager

### ISSUE THREAD

**Issue by erh3cq** [hyperspy/hyperspyUI#140]
Click on settings->plugin manager and get the below error.  Seems to dislike QT4, is there a way around this?  I am running pyqt 4.11. I can not open the UI with the newest 5. version.
```
Traceback (most recent call last):
  File "c:\users\ericho~1\documents\github\hyperspyui\hyperspyui\mainwindow.py", line 375, in show_plugin_manager
    self.plugin_manager, self)
  File "c:\users\ericho~1\documents\github\hyperspyui\hyperspyui\widgets\pluginmanagerwidget.py", line 131, in __init__
    self.create_controls()
  File "c:\users\ericho~1\documents\github\hyperspyui\hyperspyui\widgets\pluginmanagerwidget.py", line 152, in create_controls
    h.setResizeMode(QHeaderView.ResizeToContents)
  File "C:\Users\Eric Hoglund\Anaconda3\envs\data-analysis\lib\site-packages\qtpy\_patch\qheaderview.py", line 83, in setResizeMode
    raise Exception('setResizeMode is only available in Qt4. Use '
Exception: setResizeMode is only available in Qt4. Use setSectionResizeMode instead.
```

**Comment by vidartf** [comment]
What is your version of qtpy (not pyqt)? Considering that the error comes from that package, I would say it is the most likely suspect.

**Comment by erh3cq** [comment]
1.2.1, showing that it is the most current version

**Comment by vidartf** [comment]
Actually, I don't understand why qtpy gets imported at all. Is this a dependency of hyperspy now?

**Comment by erh3cq** [comment]
It is not a dependency of hyperSpy.  The third to last line suggests HyperspyUI is calling qtpy:
`\hyperspyui\widgets\pluginmanagerwidget.py", line 152, in create_controls
    h.setResizeMode(QHeaderView.ResizeToContents)`

**Comment by vidartf** [comment]
The UI definitely uses PyQt, but as far as I know, nothing depends on qtpy. Maybe you have a plugin that import qtpy?

**Comment by vidartf** [comment]
Is this resolved after #150?

**Comment by ericpre** [comment]
Fixed in #162.
